examples/DotNetCore/WebDemo/Microsoft.Azure.AppConfiguration.WebDemo/Startup.cs

```diff
@@ -14,34 +15,42 @@ public Startup(IConfiguration configuration)
 
         public IConfiguration Configuration { get; }
 
+        // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
-            //
+            services.AddRazorPages();
+
             // We add a Settings model to the service container, which takes its values from the applications configuration.
             services.Configure<Settings>(Configuration.GetSection("WebDemo:Settings"));
-
-            services.AddMvc();
         }
 
-        public void Configure(IApplicationBuilder app, IHostingEnvironment env)
+        // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
+        public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
         {
+            // Enable automatic configuration refresh from Azure App Configuration
+            app.UseAzureAppConfiguration();
```

Comment:
Might be better to have middlewares being registered after exception handler/page.

Reply:
Good suggestion. Updated.